Dockerfile.mips64le

```diff
@@ -0,0 +1,13 @@
+FROM loongnix/alpine:3.11
```

Comment:
Please use `mips64le/debian:stable-20211220-slim` to match what we use in flannel's cni-plugin for [mips64le](https://github.com/flannel-io/cni-plugin/blob/main/Dockerfile.mips64le).

Reply:
OK, i have changed the image and modify the relevant package. If there is no problems, i will delete the annotations.